aries_cloudagent/protocols/present_proof/v1_0/manager.py

```diff
@@ -294,6 +294,21 @@ async def create_presentation(
                     await holder.get_credential(credential_id)
                 )
 
+        for cred_id in credentials:
+            if not credentials[credential_id].get("rev_reg_id"):
```

Comment:
`credential_id` or `cred_id`?

Reply:
actually it's much worse than that - I don't know how this could possibly have worked